npe2/_from_npe1.py

```diff
@@ -481,7 +485,7 @@ def _write_new_setup_cfg_ep(info: PluginPackage, mf_name: str):
     mf_path = f"{info.top_module}:{mf_name}"
     new_ep = f"\n{info.package_name} = {mf_path}"
     if "options.entry_points" not in p.sections():
-        p.add_section("options.entry_points")
+        p.add_section("options.entry_points")  # pragma: no cover
```

Comment:
This is logic that will get in the the "real world". So flagging this as a maybe-don't-pragma-this-one.

It doesn't really need coverage. The use of `add_section` for `package_data` is hit below.

There's sort of a goofy way to cover both by factoring a method gets reused for lines on 487 and 491.

Reply:
added a suggestion showing what I mean. I'm ok with the pragma. just pointing this out.

Reply:
> This is logic that will get in the the "real world". 

actually, this conditional will probably _never_ hit.  The reason is that `_write_new_setup_cfg_ep` only gets called if the plugin is using `setup.cfg` (instead of `setup.py`), and if they don't already have an `entry_points` section in the `setup.cfg`, then they're effectively not a plugin anyway, and it wouldn't have made it to this point.  We could even just delete this conditional, but it seems safer to include, but not worth testing